tests/test_pacer.py

```diff
@@ -543,7 +543,8 @@ class DocketParseTest(unittest.TestCase):
     def setUp(self):
         self.maxDiff = 200000
 
-    def run_parsers_on_path(self, path_root):
+    def run_parsers_on_path(self, path_root,
+    requiredFields = ['date_filed', 'case_name', 'docket_number']):
```

Comment:
Make this `required_fields` — snake case in Python, please. Also, the indentation and spacing here seem wonky.

Reply:
Whoops...too many languages.